bears/css/CSSCombBear.py

```diff
@@ -0,0 +1,226 @@
+                        sort_order_fallback: bool = True,
+                        space_before_colon: bool = True,
+                        space_after_colon: bool = True,
+                        space_between_declaration: str = "\n",
+                        space_before_o_brace: bool = True,
+                        space_after_o_brace: bool = True,
+                        space_after_s_delim: str = "\n",
+                        space_before_c_brace: str = "\n",
+                        strip_spaces: bool = True,
+                        tab_size: bool = True,
+                        vendor_prefix_align: bool = True,
+                        csscomb_json: str = ""):
+        """
+        :param remove_empty_rulesets:
+            Remove empty rulesets if they exist.
+
+            Example:
+            .a {
+                color: tomato;
+            }
+            .b{}
+
+            After using the bear:
+            .a {
+                color: tomato;
+            }
+        :param always_semicolon:
+            If True, adds a semicolon at the end.
+
+            Example:
+            .a {
+                color: tomato
+            }
+
+            After using the bear:
+            .a {
+                color: tomato;
+            }
+        :param color_case:
+            If set to 'upper', makes all colors with upper-case letters.
+            If set to 'lower', makes all colors with lower-case letters.
+
+            Example(for upper case):
+            .a {
+                color: #fff;
+            }
+
+            After using the bear:
+            .a {
+                color: #FFF;
+            }
+        :param block_indent:
+            If set to "\t", indents by a tab.
+
+            Example:
+            .a {
+            color: tomato;
+            }
+
+            After using the bear:
+            .a {
+                color: tomato;
+            }
+        :param color_shorthand:
+            If True, changes the hexa colors into short form.
+
+            Example(for True case):
+            .a {
+                color: #ffcc00;
+            }
+
+            After using the bear:
+            .a {
+                color: #fc0;
+            }
+        :param eof_newline:
+            If True, adds a newline at EOF.
+        :param leading_zero:
+            If False, deletes the zeroes preceding the dots.
+            If True, adds the zeroes preceding the dots.
+
+            Example(False case):
+            .a {
+                padding: 0.1vh;
+                font-size: 0.5em;
+            }
+
+            After using the bear:
+            .a {
+                padding: .1vh;
+                font-size: .5em;
+            }
+        :param quotes:
+            Choose between double quotes and single quotes.
+            If "single" - single quotes.
+            If "double" - double quotes.
+        :param sort_order_fallback:
+            If True, sorts the properties alphabetically.
```

Comment:
you don't need to add ``If True`` everywhere, ``Sorts the properties alphabetically`` is enough